### PR TITLE
[WIP] dropdown: create rotating __tick to match the guide

### LINKED ISSUE #1225: dropdown: create rotating __tick element to match the guide

### PR DESCRIPTION
**Pull request by persidskiy**
close #1225

/cc @mishanga @veged @dfilatov

### CODE REVIEW

**Comment by deeonis** [comment]
@Coltspb could you plz rebase?

File: common.blocks/dropdown/dropdown.bemhtml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 block('dropdown')(
+    def()(function() {
+        applyNext({ _hasTick : this.mods['has-tick'] });
```
Comment (aristov) [inline common.blocks/dropdown/dropdown.bemhtml:3]: You can check `this.mods['has-tick']` in any template, so you shouldn't create redundant variable `_hasTick`.

File: common.blocks/dropdown/_switcher/dropdown_switcher_button.bemhtml [hunk 2]
```diff
@@ -12,6 +17,11 @@ block('dropdown').mod('switcher', 'button').elem('switcher').content()(function(
         resMods.size || (resMods.size = dropdownMods.size);
         resMods.theme || (resMods.theme = dropdownMods.theme);
         resMods.disabled = dropdownMods.disabled;
+
+        this._hasTick && this.isArray(res.content) && res.content.push({
```
Comment (aristov) [inline common.blocks/dropdown/_switcher/dropdown_switcher_button.bemhtml:21]: Let's add elem `tick` always if we get simple content. So we don't need this modifier at all.

**Comment by narqo** [comment]
@Coltspb ping!

**Comment by aristov** [comment]
@persidskiy let's clarify: are you going to do this at all??

**Comment by narqo** [comment]
Looks like nobody needs it

**Pull request closed without merging**